MAINTAINERS.yml

```diff
@@ -2903,6 +2903,10 @@ RISCV arch:
     - ycsin
   files:
     - arch/riscv/
+    - boards/enjoydigital/litex_vexriscv/
+    - boards/lowrisc/opentitan_earlgrey/
+    - boards/microchip/m2gl025_miv/
+    - boards/microchip/mpfs_icicle/
```

Comment:
do not see why those would fall under the riscv architecture, they deserver an area maintained by microchip in this case, including the socs. Architecture are should not become a collection for boards/socs of that architecture.

Reply:
Thanks for the comment. I've separated Microchip RISC-V platforms into a separate area. Please let me know if this works for you.